### ISSUE TITLE
Gang borg?

### ISSUE THREAD

**Issue by 00aquir** [yogstation13/yogstation#1911]
OOC: hey cap as to why i killed you turns out im still gang after borging
OOC: Security Cyborg-806's Memories: Objective #1: Be the first gang to successfully takeover the station with a Dominator.

Is this true?

**Comment by Super3222** [comment]
You save your memory from your old body. This includes objectives or player-submitted information as well.

While it is true that they still had that message somewhere in their memories

# they were not a gang member period. They cannot come up with an excuse like that to pull off antag-y moves as a cyborg.

So in effect they shouldn't have killed the Captain. Hope someone clears this up in a player complaint on the forums.

I might as well make cyborganization clear your notes / former objectives.

**Comment by 00aquir** [comment]
yep, but if objective memory transfers, some new players can refer to this, and if a borg gets de-brained and get a new body, will it still appear the same way?
